src/python/pants/option/global_options.py

```diff
@@ -126,6 +126,16 @@ def register_bootstrap_options(cls, register):
                   'version of the pants instance you are running using -v, -V, or --version.')
 
     register('--pants-runtime-python-version', advanced=True,
+             removal_version='1.19.0.dev0',
+             deprecation_start_version='1.17.0.dev0',
+             removal_hint="""\
```

Comment:
Preferred to use `from textwrap import dedent` so that you can ident the below text to be consistent with the rest of the block.

```python
    removal_hint=dedent("""\
      Hello
      New line""")
```

Reply:
So the reason I wanted to format it like this is because this string is going to be output to the terminal directly, and I wanted to have it look the way it would when output to the terminal, including having line widths be the same as when output to the terminal. Does that seem to be a good reason to avoid the norm here?

Reply:
I've added an initial newline so that the first line reflects the way it is output to the terminal as well.